pybossa/jobs.py

```diff
@@ -976,6 +977,27 @@ def notify_blog_users(blog_id, project_id, queue='high'):
     return msg
 
 
+def notify_project_progress(info, email_addr, queue='high'):
+    """ send email about the progress of task completion """
+
+    subject = "Project progress reminder for {}".format(info['project_name'])
+    msg = """There are only {} tasks left incompleted in your project {}.
```

Comment:
Consider rephrasing?

`There are only {} tasks left as incomplete in your project`